### ISSUE TITLE
Linux support

### ISSUE THREAD

**Issue by One-art** [kciray8/CodePaster#1]
Hi!
What about linux (Ubuntu)? I think, many developers use Linux for codding. Can you extend your app? it will be great!

Thanks

**Comment by kciray8** [comment]
Yes, now we discuss this and in new version we necessarily add linux

**Comment by tucnak** [comment]
I hope support for Linux will be added side-by-side with Mac support. If no one is  ready with that today, I am going to implement support for these platforms tomorrow.
